scripts/userstyles.yml

```
@@ -102,6 +102,9 @@ collaborators:
   - &rooot
     name: rooot
     url: https://github.com/RoootTheFox
+  - &itzTheMeow_
```

Extra underscore? Not sure why that's here...
```suggestion
  - &itzTheMeow
```

---

oops sorry, used to typing the _